docs/by-example/sqlalchemy/cru.rst

```diff
@@ -94,16 +131,32 @@ Verify the return type of date and datetime::
     >>> type(location.datetime_notz)
     <class 'datetime.datetime'>
 
-the location also has a date and datetime property which both are nullable and
-aren't set when the row is inserted as there is no default method::
+The location instance also has other ``date`` and ``datetime`` attributes which
+are nullable. Because there is no default value defined in the ORM schema for
+them, they are not set when the record is inserted:
 
     >>> location.nullable_datetime is None
     True
 
     >>> location.nullable_date is None
     True
 
-The datetime and date can be set using a update statement::
+.. hidden::
+
+    >>> from datetime import datetime, timedelta
+    >>> now = datetime.utcnow()
+
+    >>> (now - location.datetime_tz).seconds < 4
+    True
+
+    >>> (now.date() - location.date) == timedelta(0)
+    True
```

Comment:
Re. https://github.com/crate/crate-python/pull/464#discussion_r1044547187:

We still need this spot for conducting the integration tests, because we currently have no other place yet, where Python-based SQLAlchemy integration tests are conducted. This may be improved in the future. Anyhow, now those test cases are hidden from the user on the rendered documentation.

Reply:
I've created a note about this at GH-491.